pkg/cloud/gcp/gcp.go

```diff
@@ -72,11 +78,13 @@ func NewGCPProvider(
 // 1. create the inbound and outbound firewall rules for submariner, below ports will be opened
 //    - IPsec IKE port (by default 500/UDP)
 //    - NAT traversal port (by default 4500/UDP)
+//    - NAT traversal discovery port (by default 4900/UDP)
 //    - 4800/UDP port to encapsulate Pod traffic from worker and master nodes to the Submariner Gateway nodes
 // 2. create the inbound and outbound firewall rules to open 8080/TCP port to export metrics service from the Submariner gateway
 func (g *gcpProvider) PrepareSubmarinerClusterEnv() error {
-	// open IPsec IKE port (by default 500/UDP), NAT traversal port (by default 4500/UDP) and route port (4800/UDP)
-	ports := []string{g.ikePort, g.nattPort, g.routePort}
+	// open IPsec IKE port (by default 500/UDP), NAT traversal port (by default 4500/UDP),
+	// NAT Traversal discovery port (be default 4900/udp) and route port (4800/UDP)
```

Comment:
```suggestion
	// NAT Traversal discovery port (by default 4900/udp) and vx-submariner tunnel port (4800/UDP)
```

Reply:
It uses the term route port in code too, so I'd prefer to retain it to avoid confusion.

Reply:
There’s still the “be” typo ;-).